src/components/BaseModal/BaseModal.css

```
@@ -0,0 +1,46 @@
+.base-modal-container {
+  width: 100%;
+  height: 100%;
+  position: fixed;
+  top: 0;
+  left: 0;
+  display: flex;
+  z-index: 100;
+
+  justify-content: center;
+  align-items: center;
+}
+.base-modal-container--open {
+  display: flex;
+}
+.base-modal-container--closed {
+  display: none;
+}
+.base-modal-container__backdrop {
+  width: 100%;
+  height: 100%;
+  position: fixed;
+  top: 0;
+  left: 0;
+  background-color: rgba(0, 0, 0, 0.5);
+  z-index: 0;
+}
+
+.base-modal-container__modal {
+  width: 85%;
+  max-width: 480px;
+  min-width: 343px;
+  background-color: var(--color-white);
+  padding: 36px;
+  border-radius: 6px;
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+  z-index: 4;
+}
+.base-modal-container__modal__header {
```

Esta estilo se puede unir con el de abajo para hacer esta regla.
```suggestion
.base-modal-container__modal__header, .base-modal-container__modal__content {
```